### PR TITLE
feat: introduce o-normalise to o-private-foundation

### PR DESCRIPTION
**Pull request by frshwtr**
## Checklist before requesting a review

- [ ] I have applied `percy` label for o-[COMPONENT] or `chromatic` label for o3-[COMPONENT] on my PR before merging and after review. Find more details in [CONTRIBUTING.md](https://github.com/Financial-Times/origami/blob/main/CONTRIBUTING.md#pull-requests-and-visual-regression-tests)
- [ ] If it is a new feature, I have added thorough tests.
- [ ] I have updated relevant docs.
- [ ] I have updated relevant env variables in Doppler.

### CODE REVIEW

File: components/o-private-foundation/main.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import PrivateFoundation from './src/js/private-foundation.js';
```
Comment (notlee) [inline components/o-private-foundation/main.js:1]: Let's delete this and all JS

File: components/o-private-foundation/o-normalise/main.scss [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+/// @param {list} $opts - A map of which normalise styles and helper classes to output.
+/// @example Output all o-normalise styles.
+///     @include oNormalise();
+/// @example Output all o-normalise styles, without css class helpers such as `o-normalise-visually-hidden`.
+///     @include oNormalise($opts: (
+///     	'elements': ('forms', 'images', 'text', 'links'),
+///     	'body': ('font-smoothing', 'focus', 'reduce-motion')
+///     ));
+/// @access public
+@mixin oNormalise($opts: (
```
Comment (notlee) [inline components/o-private-foundation/o-normalise/main.scss:16]: As far as I know, no Origami component calls `oNormalise` so we can delete this?

Comment (frshwtr) [reply components/o-private-foundation/o-normalise/main.scss:16]: Ah yes, I assumed stories were going to be updated to use private foundation, but it doesn't seem to matter so much in this case.

File: components/o-private-foundation/o-normalise/src/scss/_mixins.scss [hunk 3]
```diff
@@ -0,0 +1,312 @@
+/// Visually hide an element while still
+/// allowing it to be read by a screenreader
+@mixin oNormaliseVisuallyHidden {
```
[the remark this answers is not in the document]
Comment (frshwtr) [reply components/o-private-foundation/o-normalise/src/scss/_mixins.scss:3]: 👍 I will prefix any of these remaining functions with `oPrivate` and kept the rest of the name the same.

File: components/o-private-foundation/o-normalise/main.scss (hunk 2 above)
Comment (notlee) [reply components/o-private-foundation/o-normalise/main.scss]: Yeah let's exclude demos

File: components/ft-concept-button/main.scss [hunk 4]
```diff
@@ -1,5 +1,5 @@
 @import '@financial-times/o-typography/main';
-@import '@financial-times/o-normalise/main';
+@financial-times/o-normalise/main
```
Comment (notlee) [inline components/ft-concept-button/main.scss:2]: oops

File: components/o-private-foundation/src/tsx/private-foundation.tsx [hunk 5]
```diff
@@ -0,0 +1,3 @@
+export function PrivateFoundation() {
+	return (<div className="o-private-foundation"data-o-component="o-private-foundation"></div>)
```
Comment (notlee) [inline components/o-private-foundation/src/tsx/private-foundation.tsx:2]: We don't need this.

File: components/o-autocomplete/main.scss [hunk 6]
```diff
@@ -1,10 +1,11 @@
+@use '@financial-times/o-private-foundation/o-normalise/src/scss/_mixins' as normalise;
```
Comment (notlee) [inline components/o-autocomplete/main.scss:1]: still using dart modules here

File: components/o-typography/stories/typography.scss [hunk 7]
```diff
@@ -1,5 +1,5 @@
 @import "@financial-times/o-fonts/main";
-@import "@financial-times/o-normalise/main";
+@import "@financial-times/o-private-foundation/o-normalise/main";
```
Comment (notlee) [inline components/o-typography/stories/typography.scss:2]: We should leave `o-normalise` as a dev dependency for stories and demos, so they can use the full range of mixins including `oNormalise`

File: components/o-private-foundation/o-normalise/src/scss/_variables.scss [hunk 8]
```diff
@@ -0,0 +1,8 @@
+@import "@financial-times/o-colors/main";
+/// Silent mode variable
+$o-normalise-is-silent: true !default;
```
Comment (notlee) [inline components/o-private-foundation/o-normalise/src/scss/_variables.scss:3]: This variable isn't needed

File: components/o-private-foundation/o-normalise/src/scss/_variables.scss [hunk 9]
```diff
@@ -0,0 +1,8 @@
...
+
+/// @access private
+$_o-normalise-focus-color: oColorsByUsecase('focus', 'outline', $fallback: null);
+$_o-normalise-focus-ring: 0 0 0 4px oColorsByName('white'), 0 0 0 8px oColorsByName('black-70');
```
Comment (notlee) [inline components/o-private-foundation/o-normalise/src/scss/_variables.scss:7]: Sorry I didn't think before: These need to be namespaced also, they will clash with o-normalise currently

**Pull request merged**